include/ocpp/v201/database_handler.hpp

```diff
@@ -118,6 +118,9 @@ class DatabaseHandler : public common::DatabaseHandlerCommon {
 
     /// \brief Remove all metervalue entries linked to transaction with id \p transaction_id
     bool transaction_metervalues_clear(const std::string& transaction_id);
+
+    /// \brief Inserts or updates the given \p profile to CHARGING_PROFILES table
```

Comment:
Are we using a connector_id or an evse_id?

Reply:
Great catch. Updated to evse_id